<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HGenArray</class>
 <widget class="QDialog" name="HGenArray">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>阵面映射</string>
  </property>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>434</width>
     <height>209</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>天线阵设置 </string>
   </property>
   <widget class="QGroupBox" name="groupBox_14">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>17</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>8</y>
       <width>111</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_rect_2">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>矩形平面阵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_trigle_2">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>三角平面阵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_circle_plane_2">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>圆形平面阵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton__circle_grid_2">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>圆形栅格阵</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="m_widget_loyout_2">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>17</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="m_rectwidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QLabel" name="label_54">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>纵向阵元数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_55">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>横向阵元数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_56">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>纵向阵元间距：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_57">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>横向阵元间距：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_24">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit_y_num_rect_2">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_x_num_rect_2">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_y_range_rect_2">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_x_range_rect_2">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_25">
            <item>
             <widget class="QLabel" name="label_58">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_59">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_60">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;楷体&quot;;</string>
              </property>
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_61">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;楷体&quot;;</string>
              </property>
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="m_triangle" native="true">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>10</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>纵向阵元数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>横向阵元数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>纵向阵元间距：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_38">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>横向阵元间距：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_y_num_trigle_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_x_num_trigle_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_y_range_trigle_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_x_range_trigle_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QLabel" name="label_39">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_40">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_41">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_42">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="m_circlegrid" native="true">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>380</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>24</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_45">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>圆面半径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_46">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>径向距离：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_47">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>圆面等分：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <widget class="QLineEdit" name="lineEdit__circle_r_grid_2">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_radial_range_2">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_circle_num_grid_2">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="m_circleplane" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>114</y>
      <width>259</width>
      <height>46</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_rect_spred">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>13</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>矩形分布</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_trigle_spred">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>三角形分布</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>7</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <widget class="QLabel" name="label_48">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>圆面半径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_49">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>横向阵元间距：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_50">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>纵向阵元间距：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QLineEdit" name="lineEdit_circle_r_plane_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_x_range_plane_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_y_range_plane_2">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QLabel" name="label_51">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_52">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_53">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>222</y>
     <width>434</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;</string>
   </property>
   <property name="title">
    <string>中心偏移量</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>△X：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_offset_x"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>△Y：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_offset_y"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>△Z：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_offset_z"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
